client/asset/eth/nodeclient_harness_test.go

```diff
@@ -56,22 +59,36 @@ import (
 )
 
 const (
-	alphaNode = "enode://897c84f6e4f18195413c1d02927e6a4093f5e7574b52bdec6f20844c4f1f6dd3f16036a9e600bd8681ab50fd8dd144df4a6ba9dd8722bb578a86aaa8222c964f@127.0.0.1:30304"
-	alphaAddr = "18d65fb8d60c1199bb1ad381be47aa692b482605"
-	pw        = "bee75192465cef9f8ab1198093dfed594e93ae810ccbbf2b3b12e1771cc6cb19"
+	alphaNode         = "enode://897c84f6e4f18195413c1d02927e6a4093f5e7574b52bdec6f20844c4f1f6dd3f16036a9e600bd8681ab50fd8dd144df4a6ba9dd8722bb578a86aaa8222c964f@127.0.0.1:30304"
+	alphaAddr         = "18d65fb8d60c1199bb1ad381be47aa692b482605"
+	pw                = "bee75192465cef9f8ab1198093dfed594e93ae810ccbbf2b3b12e1771cc6cb19"
+	maxFeeRate uint64 = 200 // gwei per gas
+	// testnet can be set to true to perform tests on the goerli testnet.
+	// May need some setup including sending testnet coins to the addresses
+	// and a lengthy sync. Wallet addresses are the same as simnet. All
+	// wait and lock times are multiplied by testnetSecPerBlock. Tests may
+	// need to be run with a high --timeout=2h for the initial sync.
```

Comment:
So after the first time you run this harness, does the node data persist (and not get wiped) so another harness run can use it instead of syncing from scratch?

Reply:
Also, it would be swell if we could specify testnet bootnodes in case you have a local geth running to save internet bandwidth use.

Reply:
> So after the first time you run this harness, does the node data persist (and not get wiped) so another harness run can use it instead of syncing from scratch?

This does not use the harness at all. So yes, the folder where these nodes live persists until you delete it manually.

> Also, it would be swell if we could specify testnet bootnodes

Will add this ability.

Reply:
> This does not use the harness at all.

Right right, there's only the light nodes for the dex wallets, and those just use testnet via internet or the specified bootnode.  Got it.